src/__tests__/__snapshots__/Carousel.js.snap

```diff
@@ -204,17 +204,17 @@ exports[`Slider Snapshots center mode 1`] = `
         onTouchStart={[Function]}
         style={
           Object {
-            "MozTransform": "translate3d(NaNpx,0,0)",
```

Comment:
This happened to be NaN before because updateSizes wasn't running in test. Now it is so the values are now 0s as expected so I updated snapshot (I think that's its name!) accordingly.

Reply:
How did you update the snapshot?

Reply:
Likely the wrong way. It looked generated but I wasn't sure how to update it and was worried doing so might yank in other things and make tests pass that should fail. So I just inspected the test failure message and applied the patch manually as I considered it to be expected and correct.

Happy to revisit if needed.

Reply:
Well, tests are passing so you did well. To make sure there is nothing wrong, run `yarn update-snapshots`. If everything is correct you won't have anything to commit after that.

Reply:
Ran `yarn update-snapshots`. All tests passed and no changes reported :smile: